packages/paste-website/src/pages/foundations/elevation/index.mdx

```diff
@@ -0,0 +1,221 @@
+
+Paste's elevated background color tokens are derived from a base background color with tonal shifts to indicate elevation. Elevated surfaces darken on light backgrounds and lighten on dark backgrounds, ensuring clarity and consistency across themes.
+
+<LivePreview
+  scope={{
+    Box,
+    Text,
+  }}
+  noInline
+  showOverflow
+>
+  {ElevatedBackgroundColorTokensExample}
+</LivePreview>
+
+Elevated elements across levels that share the same background color as the main page body should use the token `$color-background-weakest`. This token matches the page body tone in light mode but appears slightly lighter in dark mode to subtly reinforce elevation.
+
+<ResponsiveImage src={ElevatedBackgroundTokens} alt="a mock comparing the 'color background weaker token' in light and dark mode used as background color for minimizable dialog" />
+
+View our full list of Elevated background color tokens
+
+<Callout variant="neutral" marginY="space70">
+  <CalloutHeading as="h4">Different uses of color</CalloutHeading>
+  <CalloutText>
+   <CalloutList as="ul">
+      <CalloutListItem>Elevation Colors: Subtle tones for prominence and hierarchy, used to separate elements visually from the background.</CalloutListItem>
+      <CalloutListItem>Semantic Colors: Used to communicate meaning or status, such as success, warning, or error.</CalloutListItem>
+      <CalloutListItem>Decorative Colors: Reflect the brand identity or give a branded feel to a composition.</CalloutListItem>
+    </CalloutList>
+  </CalloutText>
+  Tip: use elevation colors for hierarchy, semantic colors for purpose, and branded colors for identity and style.
+</Callout>
+
+## Shadow tokens
+
+Shadows are the most extreme way to display elevation. Overuse of shadows without understanding your page’s visual hierarchy can have a negative effect on the user experience.
+
+**Level 1 shadows** are accent shadows that subtly define the edges of a container. They serve as a lightweight alternative to risen shadows at lower elevations.
+
+<LivePreview
+  scope={{
+    Box,
+    Text,
+    Theme
+  }}
+  noInline
+  showOverflow
+>
+  {ShadowTokensLevel1Example}
+</LivePreview>
+
+**Level 2 shadows** are small and sharp and indicate a close distance with the surface behind them. Shadows used in dark backgrounds use a slightly darker hue so the perceived depth is consistent.
+
+<LivePreview
+  scope={{
+    Box,
+    Text,
+    Theme
+  }}
+  noInline
+  showOverflow
+>
+  {ShadowTokensLevel2Example}
+</LivePreview>
+
+**Level 3 shadows** are larger and softer, this indicates a further distance from the surface behind them. Shadows used in dark backgrounds use a slightly darker hue so the perceived depth is consistent.
+
+<LivePreview
+  scope={{
+    Box,
+    Text,
+    Theme
+  }}
+  noInline
+  showOverflow
+>
+  {ShadowTokensLevel3Example}
+</LivePreview>
+
+<Anchor href="/tokens/list#box-shadows">View our full list of Elevated shadow tokens</Anchor>
```

Comment:
suggestion: Move this to the top of the section, underneath the paragraph "Shadows are the most extreme way to display elevation. Overuse of shadows without understanding your page’s visual hierarchy can have a negative effect on the user experience."

Right now, it gets lost down here.